src/arch/arm/64/traps.S

```diff
@@ -91,8 +91,11 @@ END_FUNC(arm_vector_table)
     mrs     x21, sp_el0
     mrs     x22, ELR
     mrs     x23, SPSR
+    mrs     x24, TPIDR_EL0
+    mrs     x25, TPIDRRO_EL0
     stp     x30, x21, [sp, #PT_LR]
     stp     x22, x23, [sp, #PT_ELR_EL1]
+    stp     x24, x25, [sp, #PT_TPIDR_EL0]
```

Comment:
This won't be a 16-byte aligned write because `FaultIP` disrupts the flow, it will be faster on most machines if we move `FaultIP` lower in `enum _register`.